### PR TITLE
[TACHYON-814] Refactored more error messages

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-814

### CODE REVIEW

File: common/src/main/java/tachyon/exception/ExceptionMessage.java [hunk 1]
```diff
@@ -40,6 +40,11 @@
   TIER_ALIAS_NOT_FOUND("Tier with alias {0} not found"),
   TIER_VIEW_ALIAS_NOT_FOUND("Tier view with alias {0} not found"),
 
+  // storageDir
+  ADD_EXISTING_BLOCK("blockId {0} exists"),
```
Comment (apc999) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: for these three messages, maybe also print out which storage dir we operate on when throwing out these errors. e.g., `blockId {0} exists in {1}` and etc?

Comment (yupeng9) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: Sounds good

**Comment by apc999** [comment]
+1 LGTM

**Comment by fanshiqing** [comment]
A small patch for this PR: #1490

**Pull request merged**